src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -1005,13 +1005,14 @@ private synchronized void regenerateAITasks() {
 
 	private void enableOrDisableFields() {
 		boolean isRaider = "Raider".equals(mobBehaviourType.getSelectedItem());
-		if (isRaider) {
+		if (isRaider)
 			breedable.setSelected(false);
-		}
+
 		raidCelebrationSound.setEnabled(isRaider);
 		for (JSpinner spinner : raidSpawnsCount)
 			spinner.setEnabled(isRaider);
 		breedable.setEnabled(!isRaider);
+		aiBase.setEnabled(!isRaider);

```

Comment:
Look, you clearly did not test this.

**If I select AI base and then select Raider, this will disable AI base, which is correct, but the previously selected value will still impact the generator.**

I will need to request you to do more testing on this or I will need to terminate this PR. I can't count anymore how many times I had to retest this PR and do manual fixes due to a lack of intense testing from the author.

Reply:
(additionally don't do unrelated changes such as removing brackets or braces on PR that you previously marked as RFR)

Reply:
Ok